ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePage.tsx

```diff
@@ -61,7 +65,14 @@ export const imageDetailsQuery = gql`
     }
 `;
 
+function ScannerV4RequiredTooltip({ children }: { children: ReactElement }) {
```

Comment:
Question for understanding, `Button` below is `ReactElement` and this is local component.

For more general composition, might you use `children: ReactNode` instead?